src/mixins/filing-template-mixin.ts

```diff
@@ -132,6 +132,9 @@ export default class FilingTemplateMixin extends DateMixin {
       }
     }
 
+    // delete invalid tax id (see ticket 15122)
+    delete filing.business.taxId
```

Comment:
On line 117 we paste in the whole business object we got from the API. Now we (temporarily) remove this field that is causing the schema validation error.

Ditto for other lines below.